packages/frontend-v2/src/components/molecules/paymentCard.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { useEffect, useState } from 'react';
+import { IMembership } from '../../libs/interfaces/membership.interface';
+import { IPayment } from '../../libs/interfaces/payment.interface';
+import { planTypeMapper } from '../../services/data-mapping.service';
+import { membershipService } from '../../services/membership.service';
+import { paymentService } from '../../services/payment.service';
+import { paymentRequestService } from '../../services/paymentRequest.service';
+
+interface IProps {
+  memberShip: IMembership;
+  onPaymentReceivedFunction: CallableFunction;
+}
+
+export const PaymentCard = (props: IProps) => {
+  const { memberShip, onPaymentReceivedFunction } = props;
+  const [alreadyRequested, setAlreadyRequested] = useState<boolean>(false);
+  useEffect(() => {
+    if (memberShip.paymentRequest) setAlreadyRequested(true);
+  }, [memberShip.paymentRequest]);
+  const createPaymentRequest = async (
+    membership: IMembership
+  ): Promise<void> => {
+    if (!membership.plan) return; // TODO: how to correctly check non nullity of some fields more globally ?
+    try {
+      const newPaymentRequest = await paymentRequestService.add({
+        amount: membership.plan.price,
+        date: new Date(),
+        description: 'demandé manuellement',
```

Comment:
I propose you add a field to put the description or you can leave it empty. Otherwise we would have thousand of paymentRequest with a description of "demandé manuellement" ....